### PR TITLE
Miawong/sc 47472/admin console loads some resources from the internet

### PR DESCRIPTION
**Pull request by miaawong**
#### What type of PR is this?
type::chore

#### What this PR does / why we need it:
[Story](https://app.shortcut.com/replicated/story/47472/admin-console-loads-some-resources-from-the-internet-instead-of-kotsadm-service) 
- Monaco Editor will use local javascript files instead of requesting CDN 
- Troubleshoot icon will be retrieved from local assets folder
#### Which issue(s) this PR fixes:

Fixes #
https://app.shortcut.com/replicated/story/47472/admin-console-loads-some-resources-from-the-internet-instead-of-kotsadm-service
#### Special notes for your reviewer:

Please checkout the branch, yarn install, and run. I had to update a few packages since they were outdated. 
updated packages: 
- grafana/ui 
- pact-foundation/pact, pact-node
- monaco-editor 
- node-sass -> sass (node-sass is deprecated)
new packages: 
- grafana/data (getValueFormat function is no longer available in grafana/ui)
- file-loader (to load monaco editor) 
- monaco-editor/loader (to configure monaco editor to load js files locally) 

## Steps to reproduce

Monaco Editor 
1. Install and deploy app 
2. click on view deploy logs 
3. notice request url where js files are loaded from kotsadm not cdn by import from node modules
Screen Shot 2022-05-10 at 10 19 26 AM

Troubleshoot 
1. Navigate to Troubleshoot tab
2. Click on support bundle that has analyzer results 
3. notice that troubleshoot rocket icon was loaded from kostadm 
Screen Shot 2022-05-10 at 10 22 22 AM

#### Does this PR introduce a user-facing change?

```release-note
NONE

```

#### Does this PR require documentation?

NONE

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
   "private": true,
   "scripts": {
     "test:unit": "jest",
+    "postinstall": "cp -R node_modules/monaco-editor/min/vs/ ./public/vs",
```
Comment (miaawong) [inline web/package.json:12]: have to create copy monaco-editor files to public folder in order to load monaco-editor js files locally 
https://github.com/suren-atoyan/monaco-react/issues/168#issuecomment-762336713

**Review by rianfowler — COMMENTED** [review]
It looks like there might be unrelated changes in this PR. Are the `/kurl_proxy` or `/pkg` changes related?

File: web/webpack.config.js [hunk 2]
```diff
@@ -69,6 +69,11 @@ module.exports = function (env) {
           ],
           sideEffects: true,
         },
+        {
+          test: /\.ttf$/,
+          use: [{loader: 'file-loader'}],
+          sideEffects: true,
+        },
```
Comment (rianfowler) [inline web/webpack.config.js:76]: This is needed by the monaco editor to include its webfonts in the bundle?

**Review by rianfowler — COMMENTED** [review]
left comments on the files I reviewed. I'll test it and approve after the unrelated changes are removed from the PR.

File: web/.gitignore [hunk 3]
```diff
@@ -5,6 +5,7 @@ cypress/videos
 
 # dependencies
 /node_modules
+/public/vs
```
Comment (rianfowler) [inline web/.gitignore:8]:
```suggestion
```

probably don't need this anymore

File: web/src/components/apps/DashboardGraphsCard.jsx [hunk 4]
```diff
@@ -3,7 +3,7 @@ import dayjs from "dayjs";
 import localizedFormat from "dayjs/plugin/localizedFormat"
 import Handlebars from "handlebars";
 import Modal from "react-modal";
-import { getValueFormat } from "@grafana/ui"
+import { getValueFormat } from "@grafana/data"
```
Comment (rianfowler) [inline web/src/components/apps/DashboardGraphsCard.jsx:6]: These look like unrelated changes

**Review by rianfowler — APPROVED** [review]

File: web/src/components/apps/DashboardGraphsCard.jsx (hunk 4 above)
Comment (miaawong) [reply web/src/components/apps/DashboardGraphsCard.jsx:6]: It is related actually, when running yarn install, we were using a very outdated version of grafana/ui causing getValueFormat to not be defined. In the latest grafana/ui, we need grafana/data for the getValueFormat function. https://grafana.com/docs/grafana/latest/packages_api/data/getvalueformat/
I should've added a review comment to explain this earlier.

Comment (rianfowler) [reply web/src/components/apps/DashboardGraphsCard.jsx:6]: Gotcha thanks for that context

**Review by rianfowler — APPROVED** [review]
